compose/cli/main.py

```diff
@@ -140,6 +140,7 @@ class TopLevelCommand(DocoptCommand):
       start              Start services
       stop               Stop services
       unpause            Unpause services
+      clean              Remove orphan containers
```

Comment:
I'd like some feedback from others here. I worry that `clean` might be too generic and it won't be obvious from the name what it's supposed to do.